### DISCUSSION TITLE
Aeotech smart plug/dimmer 6 energy use not reporting

### DISCUSSION THREAD

**Discussion by brantlk** [zwave-js/zwave-js#2453] [General] [answered]
I've got Aeotech / Aeon Labs smart plug (NZW96) and dimmer (ZW099) that report energy usage. When I had these paired with the Vera the energy report was working. I switched these to home assistant / zwave JS and now the energy report is not getting updated.

**Comment by AlCalzone** [comment, marked as answer]
The discussions are not suited to adequately help you with that. Please start by opening an issue in the HomeAssistant repository, the devs will help you figure out if that's a problem in HA or node-zwave-js.
